### PR TITLE
Tiny Holoparasite ToS fix

### PR DESCRIPTION
**Pull request by CrudeWax**
## About the PR 
Fixes the bug relating to holoparasite manual/ToS display not being readable
**Screenshots**

**Changelog**

:cl: CrudeWax
- fix: fixed holoparasite ToS not being readable

### CODE REVIEW

File: Resources/Prototypes/Catalog/Fills/Paper/manuals.yml [hunk 1]
```diff
@@ -26,6 +26,8 @@
     netsync: false
     sprite: Objects/Misc/guardian_info.rsi
     state: guardian_info
+  - type: ActivatableUI
+    key: enum.PaperUiKey.Key
```
Comment (metalgearsloth) [inline Resources/Prototypes/Catalog/Fills/Paper/manuals.yml:30]: A look at ActivatableUIComponent tells me this is incorrect. https://github.com/space-wizards/space-station-14/blob/108c7cfdf6d1aa24d85ae89bc4da0078910ba4d9/Content.Server/UserInterface/ActivatableUIComponent.cs

**Review by metalgearsloth — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/Catalog/Fills/Paper/manuals.yml (hunk 1 above)
Comment (CrudeWax) [reply Resources/Prototypes/Catalog/Fills/Paper/manuals.yml:30]: Otherwise does not work.

**Comment by CrudeWax** [comment]
So what changes should be made to this?
From talks on discord it seems like currently this is just how written notes work.

**Comment by CrudeWax** [comment]
I will further open a fix solving the issue for all readable manuals

**Pull request merged**